src/algebra/group/type_tags.lean

```diff
@@ -65,6 +65,9 @@ end multiplicative
 instance [inhabited α] : inhabited (additive α) := ⟨additive.of_mul (default α)⟩
 instance [inhabited α] : inhabited (multiplicative α) := ⟨multiplicative.of_add (default α)⟩
 
+instance [h : nontrivial α] : nontrivial (additive α) := h
```

Comment:
This doesn't work if `additive` and `multiplicative` are made irreducible, which is what Gabriel is doing in a parallel PR.
Would you mind writing a slightly longer proof? (Choose an element `a` using `h`, and and then use `additive.of_mul a` as witness, see the instances above.)

Reply:
I've expanded the proofs, unfortunately `equiv.nontrivial` lives in a file that imports this one.

Reply:
Also, `additive` and `multiplicative` becoming irreducible may yet scupper other things I'm doing, even in this PR, so I should take a look at what I can do about that...

Reply:
I think in order to fully convert this PR, I'll need helper lemmas like `multiplicative.of_add.monotone`, but I'm not sure I'm the one that should be adding those...

Reply:
The rest looked okay to me